### PR TITLE
Added new guidelines for accessibility

### LINKED ISSUE #3917: [Enhance] Update Icon Page accessibility docs to Add Alt Text Guidelines for Spark
**User Story:**
As Spark we want to add Alt Text Guidelines to the Design System. 

**Notes:**
SME: Steph 
Requestor: Olya/Virginia
Brand Central decided to treat ALT text guidelines as a sub-page to the universal accessibility guidelines.

**AC:**
On https://sparkdesignsystem.com/using-spark/components/icon
- Accessibility content will be updated to the following attachment
- See PO for Page Content/Attach [Alt Text Guidelines for Spark-Accessibility docx]

**Current State:**
image

**Future State:**
image

### PR DESCRIPTION
**Pull request by jugrinovskiy**
## What does this PR do?
As Spark we want to add Alt Text Guidelines to the Design System.

### Associated Issue
Fixes #3917 

## Please check off completed items as you work.
- Accessibility content will be updated to the following attachment
- See PO for Page Content/Attach [Alt Text Guidelines for Spark-Accessibility docx]
- Content will have bullets or Headlines. (not numbers)

### Documentation
 - [ ] Update Spark Docs

### Code
 - [ ] Build Component in HTML
 - [ ] Build Component in Angular
 - [ ] Build Component in React
 - [ ] Unit Testing in HTML with `npm run test` in `html/` (100% coverage, 100% passing)
 - [ ] Unit Testing in Angular with `npm run test` in `angular/` (100% coverage, 100% passing)
 - [ ] Unit Testing in React with `npm run test` in `react/` (100% coverage, 100% passing)

### Browser Testing (current version and 1 prior)
  - [ ] Google Chrome
  - [ ] Google Chrome (Mobile)
  - [ ] Mozilla Firefox
  - [ ] Mozilla Firefox (Mobile)
  - [ ] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [ ] Microsoft Edge
  - [ ] Apple Safari
  - [ ] Apple Safari (Mobile)
  
### Accessibility Testing
  - [ ] Axe browser extension
  - [ ] Jaws Inspect
  - [ ] VoiceOver (iOS) or Talkback (Android)

### Deploy Preview Reviewed and Approved
 - [ ] Design Team
 - [ ] Accessibility Team

### Screenshots
Add screenshots to help explain your PR if you'd like. However, this is not
expected.

### CODE REVIEW

File: src/pages/using-spark/components/icon.mdx [hunk 1]
```diff
@@ -39,6 +39,56 @@ give feedback.
...
+  - Alternative text will innately improve the discoverability of your page. 
+However, always remember that the purpose of ALT text is to provide accessibility 
+for disabled users. (For example, it will empower your blind users to experience 
+the image through the use of assistive technology, such as screen readers). 
+Hence, you should never use ALT text as a SEO opportunity. 
+You can include keywords only if they help describe the image. 
+
+- Style
+  - Use sentence case. 
+  - Include a period at the end of the alt text – even if it’s not a
```
Comment (afebbraro) [inline src/pages/using-spark/components/icon.mdx:74]: Hey @jugrinovskiy ! Woot! First PR! I noticed here was to include a period at the end even its not a complete sentence but in the example up above, we have "drawing of a cat" and no period. Do we want to update our example to have one?

**Review by afebbraro — CHANGES_REQUESTED** [review]
First PR! Woot! Left some comments.

File: src/pages/using-spark/components/icon.mdx [hunk 2]
```diff
@@ -39,6 +39,56 @@ give feedback.
...
+  - Include a period at the end of the alt text – even if it’s not a 
+complete sentence. The period will provide a slight pause when 
+read by the screen reader. 
+
+- Diversity & Inclusion
+  - Always remember that your alternative text must be aligned with 
+the culture of Diversity & Inclusion. Reflective of our ISMs 
+and values, the ALT text should support and celebrate diversity 
+across all its aspects, including and not limited to race, 
+culture, disability, age, religion, gender, orientation, and
```
Comment (afebbraro) [inline src/pages/using-spark/components/icon.mdx:83]: 🏳️‍🌈  I LOVE THIS!

File: src/pages/using-spark/components/icon.mdx (hunk 1 above)
Comment (guern1kn) [reply src/pages/using-spark/components/icon.mdx:74]: **** I agree. Also, line 63 says not to include phrases like "image of" and our examples use "drawing of".

**Review by bobwise — APPROVED** [review]
Great job on this Jane!

**Review by RVMendoza — APPROVED** [review]
We've talked about this in chat before but just so it's written somewhere, I think that in the future, we should re-evaluate and put this section into it's own guide.

Some of my more editorial thoughts:
- How is this exactly relevant to icons? This is mostly about alt text on `` and not icons. As far as I can tell, our system doesn't give an opportunity to give alt text to icons. The closest thing we have is the name we give it in the system. `title="hello world"` inside of the svg code (which I don't think we've normalized or or are consistent about)
- Some of the wording can be simplified and we can reduce the use of passive voice.
- I do love the message of the final section and totally agree, but I'm not convinced of its place here. We're giving them guidance on what to do about alt text and to me it comes off as preachy. Even though it seems like our ISMs are publicly known, it's still very much inner language that we don't explain here. I suggest this entire section could be a separate page where we *really* talk about our motivations for accessibility.

I approve this because to me it's better to get this out there than to not.

**Comment by ssabo1** [comment]
Closing at this time. Will add as a guide in an upcoming sprint.

**Pull request closed without merging**